examples/widgets/chart/lv_example_chart_9.py

```diff
@@ -0,0 +1,32 @@
+import display_driver
+import lvgl as lv
+
+def add_data(t):
+    chart.set_next_value(ser, lv.rand(10, 90))
+
+    p = chart.get_point_count()
+    s = chart.get_x_start_point(ser)
+    a = chart.get_y_array(ser)
+
+    a[(s + 1) % p] = lv.CHART_POINT.NONE
+    a[(s + 2) % p] = lv.CHART_POINT.NONE
+    a[(s + 2) % p] = lv.CHART_POINT.NONE
```

Comment:
Why do you assign `lv.CHART_POINT.NONE` three times?

Reply:
i think one time is enough, but i just copied the .c example. I can adjust it though